### ISSUE TITLE
Keboard back lightiing level.

### ISSUE THREAD

**Issue by jgdub** [vinceliuice/Orchis-theme#414]
Gnome 46 
install.sh -c dark -s compact -l

Icons not visible.

![image](https://github.com/vinceliuice/Orchis-theme/assets/47904455/999c2d06-b6e4-439f-9e5a-2a799bc7a8f6)

Thank you for this great theme and your updates!

**Comment by vinceliuice** [comment]
What is this extension ?

**Comment by jgdub** [comment]
I don't believe this is an extension... Added in gnome 45. https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2820

Only use 2 extensions - User Themes, and Blur My Shell

Hardware only supports 3 levels  Off, Low and High

Dell Latitude 7400

**Comment by liznevada** [comment]
this seems to have been changed on 46.2
![image](https://github.com/vinceliuice/Orchis-theme/assets/4228154/c5db3ba6-b85d-483c-957c-586784e89778)

hardware here also only supports 3 levels.

Dell Latitude 7214 Rugged Extreme
